src/Illuminate/Contracts/Routing/UrlGenerator.php

```diff
@@ -12,7 +12,7 @@
     public function current();
 
     /**
-     * Generate a absolute URL to the given path.
+     * Generate an absolute URL to the given path.
```

Comment:
Fine. I decided to follow the convention in other comments. Also I found another details, is it correct to add them here or do I need to make a new PR?